scripts/reference.sh

```diff
@@ -20,6 +20,7 @@ echo "Aggregating oss swaggers"
 swagrag \
   -file ${CONTRACTS}/oss.yml \
   -file ${CONTRACTS}/mapsd.yml \
+  -file ${CONTRACTS}/priv/annotationd.yml \
```

Comment:
Added this here, although I'm not 100% sure what impact it has.

Reply:
this will include the annotations api in the published oss api, if the service is in, this is fine, otherwise i'd probably take this part out.